pkg/JuliaExperimental/tst/numfield.tst

```diff
@@ -13,50 +13,54 @@ gap> z:= Zero( f );;  o:= One( f );;  a:= RootOfDefiningPolynomial( f );;
 gap> v:= [ z, o ];
 [ !0, !1 ]
 gap> nv:= GAPToNemo( c, v );
-<<Julia: [0 1]>>
+<vector object of length 2 over Monoid( ... )>
 gap> mat:= [ [ o, a/2 ], [ z, o ] ];
 [ [ !1, 1/2*a ], [ !0, !1 ] ]
 gap> nmat:= GAPToNemo( c, mat );
-<<Julia: [1 1//2*a; 0 1]>>
+<matrix object of dimensions 2x2 over Monoid( ... )>
 
 #gap> JuliaTypeInfo( JuliaPointer( nmat ) );
 #"AbstractAlgebra.Generic.Mat{Nemo.nf_elem}"
 gap> PrintObj( nmat );  Print( "\n" );
 [1 1//2*a; 0 1]
-gap> IsZero( nmat );
-false
-gap> IsZero( 0 * nmat );
-true
+gap> IsZero( nmat ); # TODO/FIXME
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
```

Comment:
I guess this means there is still work we should do on the GAP side on the MatrixObj code -- and/or we should add some methods for MatrixObj here?

In any case, I put all these changes into a separate commit so that you can revert it easily.